### PR TITLE
update show donate msg logic

### PR DESCRIPTION
**Pull request by kevodwyer**

### CODE REVIEW

File: src/components/filesystem/index.js [hunk 1]
```diff
@@ -1508,7 +1513,13 @@ module.exports = {
 	},
 	
 	showDonate: function() {
-	    return this.isLoggedIn && this.usageBytes > 1024*1024 && !this.hideDonate;
+        if (! this.isLoggedIn || this.hideDonate) {
+            return false;
+        }
+        if (this.paymentProperties.paymentServerUrl != undefined && this.paymentProperties.isPaid()) {
```
Comment (ianopolous) [inline src/components/filesystem/index.js]: if we need the null check here that's a bug in paymentproperties

Comment (kevodwyer) [reply src/components/filesystem/index.js]: no bug, it is because showDonate is called before the async code in init() sets the paymentProperties.

Comment (kevodwyer) [reply src/components/filesystem/index.js]: a valid question is: do we want to still display the donate button is there is a payment server but this user has not chosen to set up a DD?

Comment (ianopolous) [reply src/components/filesystem/index.js]: I think you've made the right choice for now. Don't show it on a paid server. We can always reconsider it later.

**Pull request merged**